### DISCUSSION TITLE
docker版本建议自动生成m3u文件、txt文件等各种格式存放到指定文件夹

### DISCUSSION THREAD

**Discussion by crylcy** [Guovin/iptv-api#730] [Ideas]
大佬，是否能让docker版本自动生成m3u文件、txt文件等各种格式存放到制定文件夹并删除之前的直播源文件，这样的话，比如用kodi，每次打开自动读取这个m3u文件，就实现完全自动化，不用每次下载然后指定新文件。

**Comment by Guovin** [comment]
每次更新会自动替换旧结果文件的，这个你无需担心，你可以使用挂载将容器内的结果映射到主机，就能实现同步了。

**Reply by crylcy** [reply]
万分感谢大佬答疑解惑！
